src/platform/test/functional/apps/dashboard/group6/dashboard_snapshots.ts

```diff
@@ -71,7 +71,7 @@ export default function ({
       );
 
       await dashboard.clickExitFullScreenLogoButton();
-      expect(percentDifference).to.be.lessThan(0.022);
+      expect(percentDifference).to.be.lessThan(0.15);
```

Comment:
Temporarily updated the value here, but I'd like to update the baseline.

Existing baseline:

![es70-tsvb_dashboard-01-before](https://github.com/user-attachments/assets/fe4ac1f2-25b8-45e4-9fef-3f8422b6ff4b)

This PR:

![es70-tsvb_dashboard-02-after](https://github.com/user-attachments/assets/5aa94722-e58a-4afe-b19c-5577a182697d)

Reply:
are you still planning on updating the baseline snapshots?

Reply:
Yes, I'll do that. There was an issue with grabbing the updated baselines from CI but I'm on to fixing that.

Reply:
Reverted the assertions in 5c97a7ee866. Updated this baseline image in 41b45d1a583.